### PR TITLE
buil(deps): Bump detekt from 1.18.0 to 1.21.0

### PR DESCRIPTION
**Pull request by yeikel**

### CODE REVIEW

**Comment by oshai** [comment]
For some reason ci test didn't run on this cl, I am trying to figure out why.

**Comment by yeikel** [comment]
> For some reason ci test didn't run on this cl, I am trying to figure out why.

Thank you. I can try force-pushing a change later today to see if it works. It is possible that is currently stuck in the build queue or something failed at the GH end that retrying will fix it

**Comment by oshai** [comment]
Yes, seems it's a github issue 
image

**Comment by yeikel** [comment]
> Yes, seems it's a github issue image

It is running now

File: build.gradle.kts [hunk 1]
```diff
@@ -202,9 +207,4 @@ detekt {
     buildUponDefaultConfig = true
     config = files(rootDir.resolve("detekt.yml"))
     parallel = true
-
-    reports {
-        html.enabled = false
-        txt.enabled = false
-    }
 }
```
Comment (yeikel) [inline build.gradle.kts:210]: See https://github.com/detekt/detekt/pull/3687

**Comment by oshai** [comment]
Is it ready for merge?

**Pull request closed without merging**